identity/migrations/2020-11-02-210000_add_unique_roles/up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE roles ADD UNIQUE (name);
```

Comment:
Not too sure about it, but shouldn't you also add the data type of the `name`s column?

Reply:
Ah, ohh, I see.. I misunderstood what it does. Since the column already exists and you are just adding the `UNIQUE` it's totally fine.